Script/char.dat

```diff
@@ -2599,6 +2893,9 @@ hunter
       "\"It wasn't my fault. That bastard had it coming. He should have remembered the rules: No kill stealing!\"",
```

Comment:
Yeah, I'll check. There must be something weird with my editor, because it occasionally crops up around apostrophes.